core/src/main/scala/org/apache/spark/ExecutorAllocationManager.scala

```diff
@@ -581,7 +581,10 @@ private[spark] class ExecutorAllocationManager(
       if (decommissionEnabled) {
         val executorIdsWithoutHostLoss = executorIdsToBeRemoved.toSeq.map(
           id => (id, ExecutorDecommissionInfo("spark scale down"))).toArray
-        client.decommissionExecutors(executorIdsWithoutHostLoss, adjustTargetNumExecutors = false)
+        client.decommissionExecutors(
+          executorIdsWithoutHostLoss,
+          adjustTargetNumExecutors = false,
+          decommissionFromDriver = true)
```

Comment:
I think decommissionFromDriver = false means that decommission is triggered by a SIGPWR on the executor. 

Whereas, decommissionFromDriver = true means that it might be triggered form either the Master or the Dynamic allocation manager. 

If so, should we flip decommissionFromDriver and just call it decomTriggeredByExecutor ?

Reply:
Let's be consistent with Master and Worker: `triggeredByExecutor`?